app/api/currentuser/delete.feature

```diff
@@ -0,0 +1,143 @@
+Feature: Delete the current user
+  Background:
+    Given the database has the following table 'groups':
+      | ID | sType     | sName      |
+      | 1  | Base      | Root       |
+      | 2  | Base      | RootSelf   |
+      | 3  | Base      | RootAdmin  |
+      | 4  | Base      | RootTemp   |
+      | 21 | UserSelf  | user       |
+      | 22 | UserAdmin | user-admin |
+      | 31 | UserSelf  | tmp-1234   |
+    And the database has the following table 'groups_groups':
+      | idGroupParent | idGroupChild |
+      | 1             | 2            |
+      | 1             | 3            |
+      | 2             | 4            |
+      | 2             | 21           |
+      | 3             | 22           |
+      | 4             | 31           |
+    And the database has the following table 'groups_ancestors':
+      | idGroupAncestor | idGroupChild | bIsSelf |
+      | 1               | 1            | true    |
+      | 1               | 2            | false   |
+      | 1               | 3            | false   |
+      | 1               | 4            | false   |
+      | 1               | 21           | false   |
+      | 1               | 22           | false   |
+      | 1               | 31           | false   |
+      | 2               | 2            | true    |
+      | 2               | 4            | false   |
+      | 2               | 21           | false   |
+      | 2               | 31           | false   |
+      | 3               | 3            | true    |
+      | 3               | 22           | false   |
+      | 4               | 4            | true    |
+      | 4               | 31           | true    |
+      | 21              | 21           | true    |
+      | 22              | 22           | true    |
+      | 31              | 31           | true    |
+    And the database has the following table 'users':
+      | ID | tempUser | sLogin   | idGroupSelf | idGroupOwned | loginID |
+      | 11 | 0        | user     | 21          | 22           | 1234567 |
+      | 12 | 1        | tmp-1234 | 31          | null         | null    |
+    And the application config is:
```

Comment:
If I check in "RemoveUserClass", there were also "users_answers", "users_items", "groups_items_propagate", ... included in the deletion. I would expect to see them deleted and included at least in one test.

Reply:
TestUserStore_DeleteWithTraps() checks deletion from all the 25 tables. Are we sure we want to duplicate this whole test as a cucumber test?

Reply:
Ok, maybe not :-/